javascripts/discourse/components/modal/audio-upload.js

```diff
@@ -43,6 +45,7 @@ export default class AudioUpload extends Component {
   get recordingSize() {
     if (this._audioData) {
       let bytes = this._audioData.size;
+
```

Comment:
@Arkshine Very very minor, but try to avoid adding just standalone line breaks in a PR (as it distracts from what is being changed).

Reply:
My bad,  there was a debug here, and I forgot to remove it!